code/datums/components/bakeable.dm

```diff
@@ -67,7 +67,7 @@
 	var/atom/original_object = parent
 	var/obj/item/plate/oven_tray/used_tray = original_object.loc
 	var/atom/baked_result = new bake_result(used_tray)
-	if(baked_result.reagents) //make space and tranfer reagents if it has any
+	if(baked_result.reagents && positive_result) //make space and tranfer reagents if it has any
```

Comment:
Update comment to include what positive_result

Reply:
```suggestion
	if(baked_result.reagents && positive_result) //make space and tranfer reagents if it has any & the resulting item isn't bad food or other bad baking result
```